examples/technical_blogs/tartan_blog/build_and_run_networks.py

```diff
@@ -0,0 +1,156 @@
+import subprocess
+import os
+import shutil
+import requests
+
+# Prereqs: portBLAS, SYCL-DNN, oneapi oneapi_construction_kit should be cloned and installed.
+portBLAS_source = os.environ.get("portBLAS_source")
+SYCL_DNN_source = os.environ.get("SYCL-DNN_source")
+path_to_oneapi = os.environ.get("path_to_oneapi")
+OCK_libCL_path = os.environ.get("OCK_libCL_path")
+input_image = os.environ.get("input_image")
+
+# Set environment variables
+os.environ["CMAKE_CXX_COMPILER"] = os.path.join(path_to_oneapi, "compiler/linux/bin-llvm/clang++")
+os.environ["LD_LIBRARY_PATH"] = f"{os.path.join(path_to_oneapi, 'compiler/linux/lib/')}:{os.path.join(path_to_oneapi, 'compiler/linux/compiler/lib/intel64_lin')}:{os.environ.get('LD_LIBRARY_PATH', '')}"
+os.environ["CXX"] = os.environ["CMAKE_CXX_COMPILER"]
```

Comment:
Who's using the `CXX` and `CC` environment variables? If it's CMake, would using `CMAKE_CXX_COMPILER` and `CMAKE_C_COMPILER` work? Just seeing a potential bit of redundancy between these two sets of variables.

Reply:
CMake can either take `CC`/`CXX` from the environment, or `CMAKE_C_COMPILER`/`CMAKE_CXX_COMPILER` from the command line. A `CMAKE_CXX_COMPILER` environment variable, as far as I am aware, would not be used by anything, so the simplest way to remove redundancy would be to remove that one.